### DISCUSSION TITLE
[Feature] show in all photos button in maps view

### DISCUSSION THREAD

**Discussion by rayrelay** [immich-app/immich#13663] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

"Show in all Photos" button when selecting one image in the map view. iOS Photos does this, its very useful when searching for an image by face and then using this button to navigate to that place in the timeline where the photo was taken to see the other pictures from that shoot (because the search does not always find the one exact picture I was looking for).

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This already exists, in the dropdown menu on the asset viewer.

![image](https://github.com/user-attachments/assets/c8f72319-d696-436c-b833-b341ecac9131)
